server-common/src/test/java/org/apache/kafka/common/DirectoryIdTest.java

```diff
@@ -67,18 +68,23 @@ void testCreateAssignmentMap() {
 
     @Test
     void testIsOnline() {
+        // Given
         List<Uuid> sortedDirs = Arrays.asList(
                 Uuid.fromString("imQKg2cXTVe8OUFNa3R9bg"),
                 Uuid.fromString("Mwy5wxTDQxmsZwGzjsaX7w"),
                 Uuid.fromString("s8rHMluuSDCnxt3FmKwiyw")
         );
         sortedDirs.sort(Uuid::compareTo);
+        List<Uuid> emptySortedDirs = Collections.emptyList();
```

Comment:
Since this is a critical fix, I'm not going to hold it up for this comment. But in the future, it would be better to put separate test cases in separate test functions, rather than creating long tests.